plugin/entrySchema.go

```diff
@@ -31,6 +31,9 @@ func schema(e Entry) (*EntrySchema, error) {
 		if err != nil {
 			return nil, err
 		}
+		if graph == nil {
```

Comment:
When addressing some of the TODOs, we should add a comment to this that it's primarily to simplify testing.